helpers/error_handler/error_message.go

```diff
@@ -8,14 +8,16 @@ import (
 )
 
 const (
-	commonFieldError    = "%s is not valid"
-	emailFormatError    = "Invalid email format"
-	emailDuplicateError = "Email already exists"
-	minError            = "%s must be longer than %s"
-	passwordEqError     = "Passwords do not match"
-	requiredError       = "%s is required"
-
-	duplicateErrorCode = "23505"
+	commonFieldError        = "%s is not valid"
+	emailFormatError        = "Invalid email format"
+	emailDuplicateError     = "Email already exists"
+	minError                = "%s must be longer than %s"
+	passwordEqError         = "Passwords do not match"
```

Comment:
After some more Go conventions reading, errors should always be in lowercase: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Fixed in 43464969e9fc3212187adb73332eb81732a3e059